tests/src/test/fdc3.broadcast.ts

```diff
@@ -200,96 +216,169 @@ export default () =>
                 !contextTypes.includes("fdc3.contact") ||
                 !contextTypes.includes("fdc3.instrument")
               ) {
-                assert.fail("Incorrect context received");
+                assert.fail("Incorrect context received", errorMessage);
               } else {
                 resolve();
               }
             }
           }
+
+          //if no context received throw error
+          await wait(3000);
+          reject(
+            new Error(`${errorMessage} At least one context was not received`)
+          );
         });
       });
 
       it("Should not receive context when A & B join different user channels and app B broadcasts a listened type", async () => {
-        channelsAppContext.contextBroadcasts.contact = true;
-        //Add two context listeners to app A
-        listener = await addContextListener("fdc3.instrument");
-        listener2 = await addContextListener("fdc3.contact");
+        const errorMessage = `\r\nSteps to reproduce:\r\n- App A adds fdc3.instrument and fdc3.contact context listener\r\n- App A joins channel 2\r\n- App B joins channel 1\r\n- App B broadcasts both context types${documentation}`;
 
-        //App A joins channel 2
-        await joinChannel(2);
+        return new Promise(async (resolve, reject) => {
+          channelsAppContext.contextBroadcasts.contact = true;
 
-        //Open ChannelsApp app. ChannelsApp joins channel 1, then broadcasts both contexts
-        await window.fdc3.open("ChannelsApp", channelsAppContext);
+          //Add two context listeners to app A
+          listener = window.fdc3.addContextListener(
+            "fdc3.instrument",
+            (context) => {
```

Comment:
very minor point, the braces are not needed around the `reject` statement

Reply:
Unfortunately, the braces get re-applied automatically any time I do a rebuild